src/components/PrimarySideBar.tsx

```diff
@@ -1,27 +1,29 @@
 import { IconType } from 'react-icons';
-import { VscFiles, VscLibrary } from 'react-icons/vsc';
+import { VscFiles, VscLibrary, VscSettingsGear } from 'react-icons/vsc';
 
 import { cx } from '../cx';
 
-export type PrimarySideBarPane = 'Explorer' | 'References';
+export type PrimarySideBarPane = 'Explorer' | 'References' | '';
```

Comment:
What does empty string mean? Can you use `null` instead?

Reply:
Sure. This was a legacy implementation.